doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2048,6 +2048,10 @@ in :ref:`canonicalstructures`; here only a simple example is given.
       If a same field occurs in several canonical structures, then
       only the structure declared first as canonical is considered.
 
+   .. note::
+      To prevent a field from being involved in the inference of canonical instances,
+      its declaration can be annotated with the :g:`#[canonical(false)]` attribute.
```

Comment:
Please add an example here, because it is quite hard to understand this sentence out of context. In particular, the declaration of a field is a bit weird conceptually: it is the record which is declared.